content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx

```diff
@@ -1,16 +1,168 @@
 ---
 title: 'Create custom Prisma Client queries'
-metaTitle: 'Prisma Client extensions (Preview)'
+metaTitle: 'Prisma Client extensions: query component (Preview)'
 metaDescription: 'Extend the functionality of Prisma Client'
 tocDepth: 3
 ---
 
 <TopBlock>
 
-<Admonition type="warning">
+From version 4.7.0, you can use the `query` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to hook into the query life-cycle and modify an incoming query or its result.
```

Comment:
Suggested alternative wording around the "component", and moved the version to end of section:

```suggestion
You can use a `query` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component in an extension to hook into the query life-cycle and modify an incoming query or its result. This was first available with version 4.7.0.
```

Reply:
I've taken your suggestion, but modified the wording slightly.

Reply:
I've made a similar change to the `result` page